.github/workflows/spack_bedrock.yml

```diff
@@ -0,0 +1,30 @@
+name: spack mobject+bedrock
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore:
+    - 'doc/**'
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Spack
+      uses: actions/checkout@v2
+      with:
+        repository: spack/spack
+        path: ./spack
+    - name: Checkout mochi-spack-packages
+      uses: actions/checkout@v2
+      with:
+        repository: mochi-hpc/mochi-spack-packages
+        path: ./mochi-spack-packages
+    - name: Run a multi-line script
```

Comment:
Can you put something more descriptive for the "name:" line here so it is easier to tell at a glance what the action is doing when you browse the results?

Reply:
I changed the name line. Thank you for reviewing!